extension/js/common/api/account-servers/flowcrypt-com-api.ts

```diff
@@ -34,36 +33,34 @@ export namespace BackendRes {
 
 export class FlowCryptComApi extends Api {
 
-  public static loginWithOpenid = async (acctEmail: string, uuid: string, idToken: string): Promise<void> => {
-    const response = await FlowCryptComApi.request<BackendRes.FcAccountLogin>('account/login', {
-      account: acctEmail,
-      uuid,
-      token: null, // tslint:disable-line:no-null-keyword
-    }, undefined, { Authorization: `Bearer ${idToken}` });
-    if (response.verified !== true) {
-      throw new Error('account_login with id_token did not result in successful verificaion');
-    }
-    await AcctStore.set(acctEmail, { uuid });
-  };
-
-  public static accountUpdate = async (fcAuth: FcUuidAuth, profileUpdate: ProfileUpdate): Promise<BackendRes.FcAccountUpdate> => {
-    FlowCryptComApi.throwIfMissingUuid(fcAuth);
+  public static accountUpdate = async (idToken: string, profileUpdate: ProfileUpdate): Promise<BackendRes.FcAccountUpdate> => {
     return await FlowCryptComApi.request<BackendRes.FcAccountUpdate>('account/update', {
-      ...fcAuth,
       ...profileUpdate
-    });
+    }, undefined,);
```

Comment:
don't we need `idToken` here?

Reply:
We indeed need it

Reply:
Ok, looks like this functionality isn't covered with tests.
I'll add new issues for this.